tech-docs/DRIFTDETECTION.md

```diff
@@ -0,0 +1,54 @@
+# Drift Detection
```

Comment:
we could move this under [guides](https://developer.hashicorp.com/terraform/registry/providers/docs#guides)?

Reply:
Makes sense, done